doc/manual/manual.md

```diff
@@ -2521,7 +2523,8 @@ which initializes `x` to whatever it is initialized to in the C code at
 the beginning of verification. This specification is now safe for
 compositional verification: SAW won't use the specification `f_spec`
 unless it can determine that `x` still has its initial value at the
-point of a call to `f`.
+point of a call to `f`. This specification also constrains `y` to prevent
+integer overflow resulting from the `x + y` expression in `f`.
```

Comment:
It might be worth clarifying:

```suggestion
signed integer overflow resulting from the `x + y` expression in `f`,
which is undefined behavior in C.
```

(Unsigned integer overflow isn't undefined behavior, so I think it's worth clarifying which type of overflow you're referring to.)